python/cudf/cudf/tests/test_repr.py

```diff
@@ -1488,7 +1488,7 @@ def test_large_unique_categories_repr():
     pi = pd.CategoricalIndex(range(100_000_000))
     gi = cudf.CategoricalIndex(range(100_000_000))
     expected_repr = repr(pi)
-    with utils.cudf_timeout(2, timeout_message="Failed to repr fast enough"):
+    with utils.cudf_timeout(6):
```

Comment:
Is there a good way to decide what this timeout should be?

Reply:
Ideally 2 seconds on my workstation, and 3x on CI. The bug was on the order of 20-30 seconds+.